tests/navier_stokes/poiseuille_flow_2d.input

```diff
@@ -0,0 +1,179 @@
+// physical parameters
```

Comment:
Suggest adding the mpi version of the input and output file. This will run and check the parallel version of the code.